study.md

```diff
@@ -87,7 +87,9 @@ Create the following movies with the given attributes.
 | 2 | The Core | 5.4 |
 
 ```ruby
-# your answer here
+movie = Movie.create(id: 0, title: "Battlefield Earth", rating: 2.4)
```

Comment:
The `id` should be omitted.  We don't want to forcefully attempt to set `id`s on database tables that have primary key columns since they'll do it for us.